webpack/assets/javascripts/react_app/Root/ReactApp.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Router } from 'react-router-dom';
-import history from '../history';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
should this PR wait for the `connected react router`? 
https://github.com/theforeman/foreman/pull/7186

Reply:
makes sense,  but the rebase is pretty the same vice versa